lib/src/clojure_lsp/queries.clj

```diff
@@ -612,3 +615,16 @@
         (-> excluded-full-qualified-vars
             set
             (contains? fqsn)))))
+
+(defn xf-all-var-usages-to-namespaces [namespaces]
+  (comp
+    (mapcat val)
+    (filter #(identical? :var-usages (:bucket %)))
+    (filter #(contains? namespaces (:to %)))
+    (remove var-usage-from-declaration?)))
+
+(def xf-all-keyword-usages
+  (comp
+    (mapcat val)
+    (filter #(identical? :keywords (:bucket %)))
+    (remove :reg)))
```

Comment:
I liked the idea of having xfs on queries to be re-usable/composable

Reply:
Me too... I've been thinking about various ways to refactor queries to be a collection of xfs.